### ISSUE TITLE
unions with Void variants can change value through a serialization round trip

### ISSUE THREAD

**Issue by dwrensha** [kentonv/node-capnp#24]
```capnp
# test.capnp

@0xd9e3377e4e6ce441;

struct Foo {
  union {
   a @0 :Void;
   b @1 :Void;
 }
}
```

```javascript
// test.js

Capnp = require('capnp');
Foo = Capnp.import('test.capnp').Foo;

var foo = {b: false}; // any falsey value is allowed for a Void field.

var outputBuffer = Capnp.serialize(Foo, foo);
var obj = Capnp.parse(Foo, outputBuffer);

console.log(obj); // "{ b: null }", as expected

var outputBuffer2 = Capnp.serialize(Foo, obj);
var obj2 = Capnp.parse(Foo, outputBuffer2);

console.log(obj2); // "{ a: null }", not as expected; the value has changed!
```

It looks like this problem was introduced in https://github.com/kentonv/node-capnp/pull/17.

**Comment by kentonv** [comment]
Probably fixed by 10e269d.

**Comment by dwrensha** [comment]
I just confirmed that this program now has the expected output.
